dbt/parser.py

```diff
@@ -75,10 +76,19 @@ def process_refs(flat_graph):
             elif len(ref) == 2:
                 target_model_package, target_model_name = ref
 
-            target_model = dbt.utils.find_model_by_name(
-                flat_graph,
-                target_model_name,
-                target_model_package)
+            # first, look for a model that matches the current
```

Comment:
That precedence seems sane to me. I'll update this. The use case here is that a model in a package uses `{{ref('dep')}}` and then you can override dep in your project, right?

This also doesn't work quite right as I think more about it -- what you want is to continue to the next option if the one you found is disabled. @drewbanin if you have time this afternoon let's discuss